sui_core/src/authority_active/gossip/tests.rs

```diff
@@ -12,8 +12,8 @@ use crate::{
     authority_aggregator::authority_aggregator_tests::*, authority_client::NetworkAuthorityClient,
 };
 
-#[tokio::test]
-pub async fn test_gossip() {
+#[tokio::test(flavor = "current_thread", start_paused = true)]
```

Comment:
why is `flavor = "current_thread"` necessary here?

Reply:
Ah, the `start_paused`, `pause` and `advance` features of tokio testing (for time) I think require the `current_thread` runtime? (I copied it from the docs)